scripts/lib/CIME/SystemTests/dae.py

```diff
@@ -91,9 +90,12 @@ def run_phase(self): # pylint: disable=arguments-differ
                "ERROR: There were {:d} DA cycles in run but {:d} DA files were found".format(da_cycles, len(da_files) if da_files is not None else 0))
         da_files.sort()
         cycle_num = 0
+        compset = self._case.get_value("COMPSET")
+        is_dwav = '_DWAV' in compset
         for fname in da_files:
             found_caseroot = False
             found_cycle = False
+            found_signal = not is_dwav
```

Comment:
Can only do the checks if DWAV is in the compset. Currently, scripts_regression_tests uses an A compset for DAE testing.